centreon/package.json

```diff
@@ -19,124 +19,119 @@
+    "eslint-plugin-hooks": "^0.4.3",
     "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jest": "^26.0.0",
-    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-jest": "^27.1.5",
+    "eslint-plugin-jsx-a11y": "^6.6.1",
     "eslint-plugin-prefer-arrow-functions": "^3.1.4",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-react": "7.28.0",
-    "eslint-plugin-react-hooks": "^4.3.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-react": "7.31.10",
+    "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-sort-keys-fix": "^1.1.2",
     "eslint-plugin-typescript-sort-keys": "^2.1.0",
     "file-loader": "^6.2.0",
     "html-webpack-harddisk-plugin": "^2.0.0",
     "html-webpack-plugin": "^5.5.0",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^28.0.0",
-    "jest-environment-jsdom": "^28.0.0",
+    "jest": "^28.1.3",
+    "jest-environment-jsdom": "^28.1.3",
     "jest-fetch-mock": "^3.0.3",
-    "jest-junit": "^13.0.0",
-    "jsdom": "^19.0.0",
+    "jest-junit": "^14.0.1",
+    "jsdom": "^20.0.2",
     "mockdate": "^3.0.5",
-    "prettier": "^2.5.1",
+    "prettier": "^2.7.1",
     "process": "^0.11.10",
     "react-beautiful-dnd-test-utils": "^4.1.1",
     "react-mock-router": "^1.0.15",
-    "react-refresh": "^0.11.0",
+    "react-refresh": "^0.14.0",
     "resize-observer-polyfill": "^1.5.1",
     "resolve-url-loader": "^5.0.0",
-    "swc-loader": "^0.2.1",
-    "terser-webpack-plugin": "^5.3.1",
-    "typescript": "^4.6.4",
+    "swc-loader": "^0.2.3",
+    "terser-webpack-plugin": "^5.3.6",
+    "typescript": "^4.8.4",
     "url-loader": "^4.1.1",
-    "webpack": "^5.68.0",
-    "webpack-bundle-analyzer": "^4.5.0",
-    "webpack-cli": "^4.9.2",
-    "webpack-dev-server": "^4.7.4",
+    "webpack": "^5.75.0",
+    "webpack-bundle-analyzer": "^4.7.0",
+    "webpack-cli": "^4.10.0",
+    "webpack-dev-server": "^4.11.1",
     "webpack-merge": "^5.8.0"
   },
   "dependencies": {
     "@centreon/js-config": "workspace:*",
     "@centreon/ui": "workspace:*",
     "@centreon/ui-context": "workspace:*",
-    "@date-io/dayjs": "^2.13.1",
+    "@date-io/dayjs": "^2.16.0",
     "@dnd-kit/core": "^6.0.5",
     "@dnd-kit/sortable": "^7.0.1",
     "@dnd-kit/utilities": "^3.2.0",
-    "@emotion/react": "^11.7.1",
-    "@emotion/styled": "^11.6.0",
-    "@mui/icons-material": "^5.3.1",
-    "@mui/lab": "^5.0.0-alpha.63",
-    "@mui/material": "^5.4.0",
-    "@mui/styles": "^5.3.0",
-    "@mui/system": "^5.10.12",
-    "@mui/x-date-pickers": "^5.0.6",
-    "@tanstack/react-query": "^4.1.0",
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
+    "@mui/icons-material": "^5.10.9",
+    "@mui/lab": "^5.0.0-alpha.107",
+    "@mui/material": "^5.10.13",
+    "@mui/styles": "^5.10.10",
+    "@mui/system": "^5.10.13",
+    "@mui/x-date-pickers": "^5.0.8",
+    "@swc/plugin-emotion": "^2.5.21",
+    "@tanstack/react-query": "^4.14.6",
+    "@tanstack/react-query-devtools": "^4.14.6",
```

Comment:
shouldn't this be a devDependency?

Reply:
No, it's excluded in the bundle so we can install it as a dependency: https://tanstack.com/query/v4/docs/devtools#install-and-import-the-devtools